__tests__/api-test.test.js

```diff
@@ -26,3 +29,14 @@ describe("/api/topics", () => {
   });
 });
 
+describe("/api", () => {
+  test("GET:200 sends object describing all the available endpoints in API", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toEqual(endpointObj);
+        expect(Object.keys(body).length).toBe(Object.keys(endpointObj).length);
```

Comment:
Line 39 isn't necessarily adding anything to your test here. The previous line makes sure that the content is what you expect it to be